### ISSUE TITLE
Add optional rangeKey support

### ISSUE THREAD

**Issue by jepetko** [tristanls/dynamodb-lock-client#25]
add optional rangeKey support so that the locking can be used like this:

```javascript
            failOpenClient.acquireLock([LOCK_ID, SUB_LOCK_ID], (error, lock) => {
                if (error) {
                    reject(error);
                } else {
                    console.log(`acquired open lock with fencing token ${lock.fencingToken}`);
                    lock.on('error', () => console.error('failed to heartbeat!'));
                    resolve(lock);
                }
            });
```

where
* `LOCK_ID` is the value of the configured partitionKey
* `SUB_LOCK_ID` is the value of the configured rangeKey

**Comment by jepetko** [comment]
@tristanls  I implemented support for range key and would like to push into a branch and create PR.
However, if I try to do it, I'm receiving 403:

```
remote: Permission to tristanls/dynamodb-lock-client.git denied to jepetko.
fatal: unable to access 'https://jepetko@github.com/tristanls/dynamodb-lock-client.git/': The requested URL returned error: 403
```

Do you have to grant me some permissions to do so? Thank you.

**Comment by tristanls** [comment]
Permission to push a branch is not required for pull requests. A workflow I
am familiar with would be to fork the repository, work against the fork,
and then create a pull request from your fork.

Could you also share with me the use case for using the range key?

**Comment by jepetko** [comment]
Hi @tristanls , I created a PR. Please, review it as soon as possible. Thank you.

**Comment by tristanls** [comment]
Released [0.7.0](https://github.com/tristanls/dynamodb-lock-client/releases/tag/v0.7.0) with sort key (range key) support.
